sshd-common/src/main/java/org/apache/sshd/common/signature/Signature.java

```diff
@@ -93,4 +93,12 @@ default void update(SessionContext session, byte[] hash) throws Exception {
      * @throws Exception If failed to calculate the signature
      */
     byte[] sign(SessionContext session) throws Exception;
+
+    /**
+     * Get the original ssh name of the signature algorithm
+     * Only used for ssh cert variants, defaults to null
+     */
+    default String getSshAlgorithmName() {
+        return null;
+    }
```

Comment:
- Javadoc is not properly formatted (missing `@return`)
- Why return null and not the actual name even if not a variant ?

Reply:
Because I am too lazy to fill it for _all_ algorithms.. For most it doesn't make a difference.
But can do it for all..